app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 HM Revenue & Customs
+ * Copyright 2021 HM Revenue & Customs
```

Comment:
Minor aside that you don't have to do, but can you update the github checklist template to the smaller one we've been using?